web-wallet/src/lib/dusk/components/CopyField/CopyField.svelte

```diff
@@ -0,0 +1,52 @@
+<script>
+  import { mdiContentCopy } from "@mdi/js";
+  import { Button, Textbox } from "$lib/dusk/components";
+  import { toast } from "$lib/dusk/components/Toast/store";
+
+  /** @type {string} */
+  export let name;
+
+  /** @type {string} */
+  export let displayValue;
+
+  /** @type {string} */
+  export let rawValue;
+
+  /** @type {boolean} */
+  export let disabled = false;
+
+  function copyToClipboard() {
+    navigator.clipboard.writeText(rawValue);
```

Comment:
As the `writeText` returns a promise, we should show the success toast if it settles without errors (and and error toast if it fails). 

```js
navigator.clipboard.writeText(newValue).then(...).catch(...)
```

Reply:
Fixed.